### DISCUSSION TITLE
Unneccesary and broken code block if i think

### DISCUSSION THREAD

**Discussion by devhiko** [twbs/bootstrap#37036] [Q&A] [answered]
There a codeblock like so and not working and throwing error when importing in a custom .scss file as shown below, please help thanks.

![error](https://user-images.githubusercontent.com/99843102/186994079-ee0c0216-fa6a-4efa-a0ac-76c4c675376c.png)

**Comment by mdo** [comment, marked as answer]
https://github.com/twbs/bootstrap/issues/37037#issuecomment-1229147857
